src/lib/list/selection-list.ts

```diff
@@ -334,4 +294,9 @@ export class MdSelectionList extends _MdSelectionListMixinBase
   private _isValidIndex(index: number): boolean {
     return index >= 0 && index < this.options.length;
   }
+
+  /** Returns the index of the specified list option. */
+  private _getIndexFromOption(option: MdListOption): number {
```

Comment:
Call this something like `_getOptionIndex`?

Reply:
Sounds also good